content/en/docs/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-23-2.md

```diff
@@ -0,0 +1,161 @@
+---
+title: v2.23.2 Armory Release (OSS Spinnaker™ v1.23.5)
+toc_hide: true
+version: 02.23.02
+description: >
+  Release notes for the Armory Platform
+---
+
+## 2020/12/14 Release Notes
+
+> Note: If you're experiencing production issues after upgrading Spinnaker, rollback to a [previous working version]({{< ref "upgrade-spinnaker#rolling-back-an-upgrade" >}}) and please report issues to [http://go.armory.io/support](http://go.armory.io/support).
+## Required Halyard or Operator version
+
+To install, upgrade, or configure Armory 2.23.2, use one of the following tools:
+
+- Armory-extended Halyard 1.10 or later
+- Armory Operator 1.2.1 or later
+
+## Security
+
+Armory scans the codebase as we develop and release software. Contact your Armory account representative for information about CVE scans for this release.
+
+## Breaking changes
+<!-- Copy/paste from the previous version if there are recent ones. We can drop breaking changes after 3 minor versions. Add new ones from OSS and Armory. -->
+
+{{< include "bc-docker-giduid.md" >}}
+
+{{< include "bc-k8s-job-suffix.md" >}}
+
+## Known issues
+<!-- Copy/paste known issues from the previous version if they're not fixed. Add new ones from OSS and Armory. If there aren't any issues, state that so readers don't think we forgot to fill out this section. -->
+
+- The Packer version included with Rosco disregards package overrides that use the `-var-file=` option.
```

Comment:
```suggestion
- The Packer version included with Rosco disregards package overrides that use the `-var-file=` option. This may cause multi-region bakes to fail.